app/models/tag.rb

```diff
@@ -36,6 +36,22 @@ class Tag < ActsAsTaggableOn::Tag
     searchableAttributes %w[name short_summary]
   end
 
+  def index_to_elasticsearch
+    Search::TagEsIndexWorker.perform_async(id)
+  end
+
+  def index_to_elasticsearch_inline
+    Search::Tag.index(id, serialized_search_hash)
```

Comment:
while revewing this methods manually, I noticed how this returns stuff, sort of diagnostics data, do we care about the result?

Reply:
For now, we are not using this data but down the road if we decide to use say ES generated ID this might be the spot where we record that. This is another instance of I choose to leave the behavior as is and kinda see where it leads us.